views/index.pug

```diff
@@ -0,0 +1,22 @@
+html
+  include partials/head
```

Comment:
you named your folder partial not partials... which is why your code doesnt work